### DISCUSSION TITLE
Changing step number for dot Navigation in reactour v1

### DISCUSSION THREAD

**Discussion by harsz** [elrumordelaluz/reactour#585] [Q&A]
Hi, thank you for such an amazing library !

Currently I am using reactour v1  to setup tour on various routes in app. Tour navigates to next view n tour continues, it works well as I have overwritten step number on next view(instead of showing step 1 on new view, tour has to be in flow with previous view, hence programmatically overwritten step no so it doesn't start with 1 on next view). Issue is dot navigation, when hovered over still have step number started from 1, 2 n so on.
Is there any way to customize step number for dot Navigation?

**Comment by elrumordelaluz** [comment]
Hi @harsz, thanks for open the _discussion_.

Mind creating a minimal reproduction in a sandbox in order to allow others to debug your use-case and try to find a solution? Thanks!
